app/scripts/views/forms.coffee

```diff
@@ -971,6 +972,22 @@ define [
       @listenToNewFormView()
       @showNewFormViewAnimate()
 
+
+    # Request the history of the form model, the user can click on a revision and see the details
+    # preferably by simply showing the other revision as a model along side in the same views
+    fetchHistory: (formModel) ->
+
+      var fielddbHelperModel = new FieldDB.Datum({_id: formModel.id});
+      fielddbHelperModel.fetchRevisions().then(function(revisions){
+        formModel.set("history", revisions.map(function(revisionUrl){
+          # TODO decide what data representation you want for these
+          return {url: revisionUrl};
+      }, function(error){
```

Comment:
thanks, i had found that already. i had a new question for you. (you have to open the PR on github in order to see the current state)